communications.md

```diff
@@ -2,95 +2,62 @@
 
 ## General guidelines
 
-Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. Therefore, the focus should be on real problem solving and creative endeavors instead of drowning oneself in processes and tools when communicating with one another.
+Communication is critical to any organization and in doing so, we should be considerate of other people's production processes and time.
+We use a few tools to aid customer and team (inter and intra) communication: Slack, G Suite, Asana and Emails.
 
-## First line of Communication: Texting
+Our communication tools encompasses both formal and informal communications as such we may want to understand:
 
-We use a few tools to help team members communicate with one another: Slack, G Suite and Asana. Before you decide to have a face-to-face or voice/video conference, take some time to organize your thoughts, and write them down. Revise it to be as concise as possible. If it's not urgently blocking all of your next tasks, use Slack, Gmail or Asana to contact another team member asynchronously (in an intermittent manner).
+- your audience 
+- the size of the audience 
+- content of message and reusability (if the content of the message will come in handy in the future) 
 
-Budget communication time. Try to allocate yourself a relatively fixed schedule to initiate, check and respond to messages. To a certain extent, this means try to optimize for the whole block of contiguous time that suits better for your job (knowledge-based, creative endeavors).
+and then choose the appropriate communication tools to ensure effective communication.
 
-#### - Internal & External Messaging Tool: Slack -
-Opt-in: Join channels that you are interested in or obligated to.
+Time management is key to success and so is effective communication. Sometimes communication tools can reduce productivity time hence, we can individually implement suitable patterns to ensure we effciently and effectively manage our time.
```

Comment:
The first sentence is a good point, and open enough. Starting from second sentence, this puts  too much emphasis on the personal pattern over a team pattern causes two issues:

1. Conflict in ways of communicating.
2. Makes the tools less productive (the very problem described in "Sometimes").

Reply:
I think time management is not just a team thing but mostly personal. Team sets out a project and delegate tasks. How individual time is managed is subjective. Provided task is completed as expected, I strongly believe time management follows individual pattern/process. 

@EQWorks/data-team @EQWorks/first-order members feel free to make suggestions on this topic so we can conclude on this

Tools can be less productive depending on usage and individual. That is why I used "sometimes". We can rephrase it as I believe it can be better written.

Reply:
> I think time management is not just a team thing but mostly personal. Team sets out a project and delegate tasks. How individual time is managed is subjective. Provided task is completed as expected, I strongly believe time management follows individual pattern/process.

you're right about time management depends on individuals. the problem of not setting a baseline for the "pattern/process" is that it's more likely to have individuals' ways conflict with each other and resulting in unnecessary counterproductive moments.

it's each individual's responsibility to manage their own time, absolutely. it's also the entire team's responsibility to eliminate as many common frictions that could result in less effective time management for most/all members of the team. this sticks to the summary line of this repo

> Common practices and disciplines that help to minimize common obstacles and frictions for a good work culture and environment.

Reply:
I feel that both arguments here is highly opinionated. It is very hard to generalize across 15+ people. What if we just do a separate **best practices** markdown page and link it here? This way, we can list out a couple of strategies and add to it as different ways of working emerge.

At the end of the day, we want to empower the team member and educate them on options open to them when it comes to various tools we have available.

I think reasonable response time, availability listing, notifications etc. can be common team-wide (the bare minimum for frictionless collab) and we can make the rest best practices (with more than one option) for individuals to choose from.